common/plugins/org.polarsys.capella.common.helpers/src/org/polarsys/capella/common/helpers/query/cache/QueryResultCache.java

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.common.helpers.query.cache;
```

Comment:
I would have created this class beside RepresentationHelper

Reply:
I'm not sure that org.polarsys.capella.core.data.helpers.cache.Cache meets our need. The cache has been moved to Sirius and filled in sirius-collab to limit changes to Capella and side effects.